RockWeb/Blocks/Examples/ModelMap.ascx.cs

```diff
@@ -246,10 +389,16 @@ private MClass GetPropertiesAndMethods( Type type, bool includeInherited = true
             MClass mClass = new MClass
             {
                 Name = type.Name,
+                Category = new Category
+                {
+                    Name = type.IsDefined( typeof( RockDomainAttribute ), true ) ? type.GetCustomAttribute<RockDomainAttribute>( true ).Name : "Other",
+                    IconCssClass = type.IsDefined( typeof( RockDomainAttribute ), true ) ? type.GetCustomAttribute<RockDomainAttribute>( true ).IconCssClass : "fa fa-file-o"
+                },
```

Comment:
Can you do this in a way where you don't have to call IsDefined and GetCustomAttribute twice for each model.  Something like:
```
var attr = type.GetCustomAttribute<RockDomainAttribute>()
...
Category = new Category
{
    Name = attr != null ? attr.Name : "Other";
    IconCssClass = attr != null ? attr.IconCssClass : "fa fa-file-o";
}
```